dali/operators/imgcodec/image_decoder.h

```diff
@@ -79,6 +79,8 @@ static constexpr size_t kHostAlignment = 64;  // cache alignment
 inline int static_dali_device_malloc(void *ctx, void **ptr, size_t size, cudaStream_t stream) {
   auto *mr = static_cast<mm::device_async_resource *>(ctx);
   try {
+    std::cout << "device_malloc this_thread[" << std::this_thread::get_id() << "] stream=" << stream
+              << "\n";
```

Comment:
Is this intended or it snuck from some debugging code?

Reply:
removed it